quotes.txt

```diff
@@ -14,3 +14,39 @@ Windows is "I'm locking my sandwich in this safe. Why are you
 worried it will go stale?"
 
     --  pjf, IRC
+%
+Jenkins is refusing to work a simple fix and keeps reporting the
+build as broken. 3 different people are trying to figure out why
+NetKAN reports an error and calls a forth one to come check it out.
+
+"licence" is misspelled
+
```

Comment:
Strictly speaking, that wasn't a typo. It was an alternate spelling, perfectly correct linguistically, but wrong programatically